Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM dwolla/jenkins-agent-nvm
+ARG CYPRESS_TAG
```

Comment:
This is kind of nitty, but should the arg be NVM_TAG?

Reply:
good call